### ISSUE TITLE
What loss function is optimized in TextCategorizer?

### ISSUE THREAD

**Issue by ashimko** [explosion/spaCy#1869]
Hello! 
Say me please, how can I choose or define loss function for TextCategorizer?
I need optimize column-wise (the score is the average of the log loss of each predicted column) log loss in multiclass classification.
Thank you.

**Comment by honnibal** [comment]
The `TextCategorizer` uses multi-label log loss. The logistic function is applied to each neuron in the output layer independently. This is similar to the normal softmax, except the scores are not normalised.

Here is the function where the model is defined: https://github.com/explosion/spaCy/blob/master/spacy/_ml.py#L469

I'm not sure whether the current behaviour matches what you want.

**Comment by ashimko** [comment]
Ok, thak you for explanation. I can not set my own metric, can I?
